src/components/light.js

```diff
@@ -281,7 +281,11 @@ module.exports.Component = registerComponent('light', {
     var distance = data.distance;
     var groundColor = new THREE.Color(data.groundColor);
     groundColor = groundColor.getHex();
-    var intensity = data.intensity;
+    // WebGLRenderer.useLegacyLights has been removed in THREE r165
+    // Lights intensity had an implicit scaleFactor of PI now removed.
+    // It's reintroduced here since it's easier to think about integers vs multiples of PI.
+    var intensityScaleFactor = Math.PI;
+    var intensity = data.intensity * intensityScaleFactor;
```

Comment:
It shouldn't change the intensity for scene that used `renderer="physicallyCorrectLights: true"` already.

Reply:
Not sure I understand the comment. Without the factor scene is dark. What should we do?

Reply:
We have aframe experiences with renderer="physicallyCorrectLights:true" and light intensity that was carefully set, those scenes are not dark, we don't want the intensity to change for those.